Windows installation of Swift 5.9 is suddenly failing without much diagnostic info.

- [x] I have gone through the action [README](https://github.com/SwiftyLab/setup-swift/blob/main/README.md), [documentation](https://github.com/SwiftyLab/setup-swift/blob/main/action.yml) and [wiki](https://github.com/SwiftyLab/setup-swift/wiki).
- [x] I don't see my issue listed in [existing issues](https://github.com/SwiftyLab/setup-swift/issues).

**Describe the bug**
Getting installed Swift version swift-5.9.2-RELEASE
C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swift.exe --version
Error: The process 'C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swift.exe' failed with **exit code 3221225781**

**To Reproduce**
using the `windows-latest` runner,

```
    - name: 'Set up swift'
      uses: SwiftyLab/setup-swift@latest
      with:
        swift-version: 5.9
```

**Logs**
Please see [this link](https://github.com/hylo-lang/Swifty-LLVM/actions/runs/8476918338/job/24227110649#step:3:723).

This is a re-run of a CI run that was successful a few weeks ago, so it should have succeded..

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Runner (please complete the following information):**
 - Name: windows-latest
 - OS: windows-2022
 - Version 20240421.1.0

Everything google tells me about **exit code 3221225781** seems to indicate there is a missing Microsoft VC++ Redistributable.

OK, this is really weird.  I have exactly the same action succeeding on `windows-latest` in a different project.  When I diff the logs for the setup swift step in these two projects, they are largely identical, but one thing stands out: when this command executes:
```
[command]C:\Windows\system32\cmd.exe /k "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\VsDevCmd.bat" -arch=x64 && set && exit
```
It dumps the environment, and the `Path` variables are different.  In the case where the action works, there is one additional entry in the `Path` that is not present where the action fails:

```
C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Swift\runtime-development\usr\bin
```

The complete diff of the two logs, in case I've overlooked something else that's important, is here:

```diff
--- pass	2024-04-25 13:33:32.115041407 -0700
+++ fail	2024-04-25 13:55:39.409608585 -0700
@@ -6,10 +6,8 @@
   dry-run: false
   cache-snapshot: true
 env:
-  spm-build-options: -Xswiftc -enable-testing --explicit-target-dependency-import-check error
-  spm-test-options: --parallel
   swift-version: 5.9
-  llvm_url_prefix: https://github.com/hylo-lang/llvm-build/releases/download/20240303-215025
+  url_prefix: https://github.com/hylo-lang/llvm-build/releases/download/20240303-215025
   llvm_package_basename: llvm-17.0.6-x86_64-unknown-windows-msvc17-MinSizeRel
 ##[endgroup]
 ##[group]Syncing swift.org data
@@ -25,13 +23,11 @@
 Detected windows platform with arch "x86_64"
 ##[endgroup]
 ##[group]Installing Swift toolchain snapshot swift-5.9.2-RELEASE
-Received 218103808 of 954751992 (22.8%), 207.0 MBs/sec
-Received 562036736 of 954751992 (58.9%), 267.2 MBs/sec
-Received 754974720 of 954751992 (79.1%), 239.0 MBs/sec
-Received 939524096 of 954751992 (98.4%), 222.8 MBs/sec
-Cache Size: ~911 MB (954751992 B)
-[command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/675dc29c-5292-4301-bb22-f92c44c25acc/cache.tzst -P -C D:/a/hylo/hylo --force-local --use-compress-program "zstd -d"
-Received 954751992 of 954751992 (100.0%), 181.3 MBs/sec
+Received 364904448 of 935859366 (39.0%), 345.6 MBs/sec
+Received 742391808 of 935859366 (79.3%), 351.5 MBs/sec
+Cache Size: ~893 MB (935859366 B)
+[command]"C:\Program Files\Git\usr\bin\tar.exe" -xf D:/a/_temp/52fc1b97-a94d-45af-bcdc-23fac3f4b64a/cache.tzst -P -C D:/a/Swifty-LLVM/Swifty-LLVM --force-local --use-compress-program "zstd -d"
+Received 935859366 of 935859366 (100.0%), 295.4 MBs/sec
 Cache restored successfully
 [command]C:\ProgramData\Chocolatey\bin\vswhere.exe -products * -format json -utf8 -latest -version 16
 [
@@ -90,11 +86,11 @@
 ** Visual Studio 2022 Developer Command Prompt v17.9.6
 ** Copyright (c) 2022 Microsoft Corporation
 **********************************************************************
-ACTIONS_CACHE_URL=https://acghubeus1.actions.githubusercontent.com/CGohErdGSQmMDBeDiiQH9snYTxvQJTv4UpfHBp5kZ3m2DZxZaI/
+ACTIONS_CACHE_URL=https://acghubeus1.actions.githubusercontent.com/EeSIM9OXx0pqtdqyY9RM6DWnEFR65ugDPiKMT45GbdUkybq6Vp/
 ACTIONS_RESULTS_URL=https://results-receiver.actions.githubusercontent.com/
 ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE=C:\actionarchivecache\
 ACTIONS_RUNTIME_TOKEN=***
-ACTIONS_RUNTIME_URL=https://pipelinesghubeus26.actions.githubusercontent.com/CGohErdGSQmMDBeDiiQH9snYTxvQJTv4UpfHBp5kZ3m2DZxZaI/
+ACTIONS_RUNTIME_URL=https://pipelinesghubeus25.actions.githubusercontent.com/EeSIM9OXx0pqtdqyY9RM6DWnEFR65ugDPiKMT45GbdUkybq6Vp/
 ALLUSERSPROFILE=C:\ProgramData
 ANDROID_HOME=C:\Android\android-sdk
 ANDROID_NDK=C:\Android\android-sdk\ndk\25.2.9519653
@@ -117,7 +113,7 @@
 CommonProgramFiles=C:\Program Files\Common Files
 CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
 CommonProgramW6432=C:\Program Files\Common Files
-COMPUTERNAME=fv-az1111-280
+COMPUTERNAME=fv-az972-783
 ComSpec=C:\Windows\system32\cmd.exe
 CONDA=C:\Miniconda
 DEPLOYMENT_BASEPATH=C:\actions
@@ -148,35 +144,35 @@
 GITHUB_ACTOR_ID=44065
 GITHUB_API_URL=https://api.github.com
 GITHUB_BASE_REF=main
-GITHUB_ENV=D:\a\_temp\_runner_file_commands\set_env_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
+GITHUB_ENV=D:\a\_temp\_runner_file_commands\set_env_f7148f03-1216-4794-ae5f-77b1afec0ccd
 GITHUB_EVENT_NAME=pull_request
 GITHUB_EVENT_PATH=D:\a\_temp\_github_workflow\event.json
 GITHUB_GRAPHQL_URL=https://api.github.com/graphql
-GITHUB_HEAD_REF=cmake
+GITHUB_HEAD_REF=needless-build-step
 GITHUB_JOB=native
-GITHUB_OUTPUT=D:\a\_temp\_runner_file_commands\set_output_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
-GITHUB_PATH=D:\a\_temp\_runner_file_commands\add_path_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
-GITHUB_REF=refs/pull/1439/merge
-GITHUB_REF_NAME=1439/merge
+GITHUB_OUTPUT=D:\a\_temp\_runner_file_commands\set_output_f7148f03-1216-4794-ae5f-77b1afec0ccd
+GITHUB_PATH=D:\a\_temp\_runner_file_commands\add_path_f7148f03-1216-4794-ae5f-77b1afec0ccd
+GITHUB_REF=refs/pull/67/merge
+GITHUB_REF_NAME=67/merge
 GITHUB_REF_PROTECTED=false
 GITHUB_REF_TYPE=branch
-GITHUB_REPOSITORY=hylo-lang/hylo
-GITHUB_REPOSITORY_ID=332354429
+GITHUB_REPOSITORY=hylo-lang/Swifty-LLVM
+GITHUB_REPOSITORY_ID=611522479
 GITHUB_REPOSITORY_OWNER=hylo-lang
 GITHUB_REPOSITORY_OWNER_ID=77444627
 GITHUB_RETENTION_DAYS=90
-GITHUB_RUN_ATTEMPT=1
-GITHUB_RUN_ID=8838325758
-GITHUB_RUN_NUMBER=3083
+GITHUB_RUN_ATTEMPT=3
+GITHUB_RUN_ID=8838213813
+GITHUB_RUN_NUMBER=329
 GITHUB_SERVER_URL=https://github.com
-GITHUB_SHA=d441884eb57491da60c008e428e1c07a24385358
-GITHUB_STATE=D:\a\_temp\_runner_file_commands\save_state_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
-GITHUB_STEP_SUMMARY=D:\a\_temp\_runner_file_commands\step_summary_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
+GITHUB_SHA=400065561505a364ee842cba37a4a4ae92a6d6f8
+GITHUB_STATE=D:\a\_temp\_runner_file_commands\save_state_f7148f03-1216-4794-ae5f-77b1afec0ccd
+GITHUB_STEP_SUMMARY=D:\a\_temp\_runner_file_commands\step_summary_f7148f03-1216-4794-ae5f-77b1afec0ccd
 GITHUB_TRIGGERING_ACTOR=dabrahams
-GITHUB_WORKFLOW=Build and test
-GITHUB_WORKFLOW_REF=hylo-lang/hylo/.github/workflows/build-and-test.yml@refs/pull/1439/merge
-GITHUB_WORKFLOW_SHA=d441884eb57491da60c008e428e1c07a24385358
-GITHUB_WORKSPACE=D:\a\hylo\hylo
+GITHUB_WORKFLOW=Test
+GITHUB_WORKFLOW_REF=hylo-lang/Swifty-LLVM/.github/workflows/test.yml@refs/pull/67/merge
+GITHUB_WORKFLOW_SHA=400065561505a364ee842cba37a4a4ae92a6d6f8
+GITHUB_WORKSPACE=D:\a\Swifty-LLVM\Swifty-LLVM
 GOROOT_1_20_X64=C:\hostedtoolcache\windows\go\1.20.14\x64
 GOROOT_1_21_X64=C:\hostedtoolcache\windows\go\1.21.9\x64
 GOROOT_1_22_X64=C:\hostedtoolcache\windows\go\1.22.2\x64
@@ -203,9 +199,8 @@
 LIB=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\lib\x64;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\lib\um\x64;C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64;C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64
 LIBPATH=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\lib\x86\store\references;C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.22621.0;C:\Program Files (x86)\Windows Kits\10\References\10.0.22621.0;C:\Windows\Microsoft.NET\Framework64\v4.0.30319
 llvm_package_basename=llvm-17.0.6-x86_64-unknown-windows-msvc17-MinSizeRel
-llvm_url_prefix=https://github.com/hylo-lang/llvm-build/releases/download/20240303-215025
 LOCALAPPDATA=C:\Users\runneradmin\AppData\Local
-LOGONSERVER=\\fv-az1111-280
+LOGONSERVER=\\fv-az972-783
 M2=C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin
 M2_REPO=C:\ProgramData\m2
 MAVEN_OPTS=-Xms256m
@@ -214,7 +209,7 @@
 npm_config_prefix=C:\npm\prefix
 NUMBER_OF_PROCESSORS=4
 OS=Windows_NT
-Path=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\VCPackages;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin\Roslyn;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\DiagnosticsHub\Collector;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Swift\runtime-development\usr\bin;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\Llvm\x64\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\Linux\bin\ConnectionManagerExe;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\vcpkg
+Path=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\VCPackages;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin\Roslyn;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\DiagnosticsHub\Collector;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\Llvm\x64\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\Linux\bin\ConnectionManagerExe;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\vcpkg
 PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
 PERFLOG_LOCATION_SETTING=RUNNER_PERFLOG
 PGBIN=C:\Program Files\PostgreSQL\14\bin
@@ -242,18 +237,16 @@
 RTOOLS44_HOME=C:\rtools44
 RUNNER_ARCH=X64
 RUNNER_ENVIRONMENT=github-hosted
-RUNNER_NAME=GitHub Actions 20
+RUNNER_NAME=GitHub Actions 17
 RUNNER_OS=Windows
 RUNNER_PERFLOG=C:\actions\perflog
 RUNNER_TEMP=D:\a\_temp
 RUNNER_TOOL_CACHE=C:\hostedtoolcache\windows
-RUNNER_TRACKING_ID=github_d03191ff-4661-46b9-aa98-c8242de0ec8c
-RUNNER_WORKSPACE=D:\a\hylo
+RUNNER_TRACKING_ID=github_f94d25e8-5833-4aca-b4b4-54d72cb05e59
+RUNNER_WORKSPACE=D:\a\Swifty-LLVM
 SBT_HOME=C:\Program Files (x86)\sbt\
 SDKROOT=C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Platforms\Windows.platform\Developer\SDKs\Windows.sdk
 SELENIUM_JAR_PATH=C:\selenium\selenium-server.jar
-spm-build-options=-Xswiftc -enable-testing --explicit-target-dependency-import-check error
-spm-test-options=--parallel
 STATS_BLT=true
 STATS_D=true
 STATS_D_D=true
@@ -272,8 +265,9 @@
 TMP=C:\Users\RUNNER~1\AppData\Local\Temp
 UCRTVersion=10.0.22621.0
 UniversalCRTSdkDir=C:\Program Files (x86)\Windows Kits\10\
-USERDOMAIN=fv-az1111-280
-USERDOMAIN_ROAMINGPROFILE=fv-az1111-280
+url_prefix=https://github.com/hylo-lang/llvm-build/releases/download/20240303-215025
+USERDOMAIN=fv-az972-783
+USERDOMAIN_ROAMINGPROFILE=fv-az972-783
 USERNAME=runneradmin
 USERPROFILE=C:\Users\runneradmin
 VCIDEInstallDir=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\
@@ -304,17 +298,17 @@
 WIX=C:\Program Files (x86)\WiX Toolset v3.14\
 __DOTNET_ADD_64BIT=1
 __DOTNET_PREFERRED_BITNESS=64
-__VSCMD_PREINIT_PATH=C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Swift\runtime-development\usr\bin;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps
+__VSCMD_PREINIT_PATH=C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps
 [command]C:\Windows\system32\cmd.exe /k "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\VsDevCmd.bat" -arch=x64 && set && exit
 **********************************************************************
 ** Visual Studio 2022 Developer Command Prompt v17.9.6
 ** Copyright (c) 2022 Microsoft Corporation
 **********************************************************************
-ACTIONS_CACHE_URL=https://acghubeus1.actions.githubusercontent.com/CGohErdGSQmMDBeDiiQH9snYTxvQJTv4UpfHBp5kZ3m2DZxZaI/
+ACTIONS_CACHE_URL=https://acghubeus1.actions.githubusercontent.com/EeSIM9OXx0pqtdqyY9RM6DWnEFR65ugDPiKMT45GbdUkybq6Vp/
 ACTIONS_RESULTS_URL=https://results-receiver.actions.githubusercontent.com/
 ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE=C:\actionarchivecache\
 ACTIONS_RUNTIME_TOKEN=***
-ACTIONS_RUNTIME_URL=https://pipelinesghubeus26.actions.githubusercontent.com/CGohErdGSQmMDBeDiiQH9snYTxvQJTv4UpfHBp5kZ3m2DZxZaI/
+ACTIONS_RUNTIME_URL=https://pipelinesghubeus25.actions.githubusercontent.com/EeSIM9OXx0pqtdqyY9RM6DWnEFR65ugDPiKMT45GbdUkybq6Vp/
 ALLUSERSPROFILE=C:\ProgramData
 ANDROID_HOME=C:\Android\android-sdk
 ANDROID_NDK=C:\Android\android-sdk\ndk\25.2.9519653
@@ -337,7 +331,7 @@
 CommonProgramFiles=C:\Program Files\Common Files
 CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
 CommonProgramW6432=C:\Program Files\Common Files
-COMPUTERNAME=fv-az1111-280
+COMPUTERNAME=fv-az972-783
 ComSpec=C:\Windows\system32\cmd.exe
 CONDA=C:\Miniconda
 DEPLOYMENT_BASEPATH=C:\actions
@@ -368,35 +362,35 @@
 GITHUB_ACTOR_ID=44065
 GITHUB_API_URL=https://api.github.com
 GITHUB_BASE_REF=main
-GITHUB_ENV=D:\a\_temp\_runner_file_commands\set_env_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
+GITHUB_ENV=D:\a\_temp\_runner_file_commands\set_env_f7148f03-1216-4794-ae5f-77b1afec0ccd
 GITHUB_EVENT_NAME=pull_request
 GITHUB_EVENT_PATH=D:\a\_temp\_github_workflow\event.json
 GITHUB_GRAPHQL_URL=https://api.github.com/graphql
-GITHUB_HEAD_REF=cmake
+GITHUB_HEAD_REF=needless-build-step
 GITHUB_JOB=native
-GITHUB_OUTPUT=D:\a\_temp\_runner_file_commands\set_output_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
-GITHUB_PATH=D:\a\_temp\_runner_file_commands\add_path_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
-GITHUB_REF=refs/pull/1439/merge
-GITHUB_REF_NAME=1439/merge
+GITHUB_OUTPUT=D:\a\_temp\_runner_file_commands\set_output_f7148f03-1216-4794-ae5f-77b1afec0ccd
+GITHUB_PATH=D:\a\_temp\_runner_file_commands\add_path_f7148f03-1216-4794-ae5f-77b1afec0ccd
+GITHUB_REF=refs/pull/67/merge
+GITHUB_REF_NAME=67/merge
 GITHUB_REF_PROTECTED=false
 GITHUB_REF_TYPE=branch
-GITHUB_REPOSITORY=hylo-lang/hylo
-GITHUB_REPOSITORY_ID=332354429
+GITHUB_REPOSITORY=hylo-lang/Swifty-LLVM
+GITHUB_REPOSITORY_ID=611522479
 GITHUB_REPOSITORY_OWNER=hylo-lang
 GITHUB_REPOSITORY_OWNER_ID=77444627
 GITHUB_RETENTION_DAYS=90
-GITHUB_RUN_ATTEMPT=1
-GITHUB_RUN_ID=8838325758
-GITHUB_RUN_NUMBER=3083
+GITHUB_RUN_ATTEMPT=3
+GITHUB_RUN_ID=8838213813
+GITHUB_RUN_NUMBER=329
 GITHUB_SERVER_URL=https://github.com
-GITHUB_SHA=d441884eb57491da60c008e428e1c07a24385358
-GITHUB_STATE=D:\a\_temp\_runner_file_commands\save_state_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
-GITHUB_STEP_SUMMARY=D:\a\_temp\_runner_file_commands\step_summary_cd2ee09d-2b21-4b44-8467-b8c9f75d405b
+GITHUB_SHA=400065561505a364ee842cba37a4a4ae92a6d6f8
+GITHUB_STATE=D:\a\_temp\_runner_file_commands\save_state_f7148f03-1216-4794-ae5f-77b1afec0ccd
+GITHUB_STEP_SUMMARY=D:\a\_temp\_runner_file_commands\step_summary_f7148f03-1216-4794-ae5f-77b1afec0ccd
 GITHUB_TRIGGERING_ACTOR=dabrahams
-GITHUB_WORKFLOW=Build and test
-GITHUB_WORKFLOW_REF=hylo-lang/hylo/.github/workflows/build-and-test.yml@refs/pull/1439/merge
-GITHUB_WORKFLOW_SHA=d441884eb57491da60c008e428e1c07a24385358
-GITHUB_WORKSPACE=D:\a\hylo\hylo
+GITHUB_WORKFLOW=Test
+GITHUB_WORKFLOW_REF=hylo-lang/Swifty-LLVM/.github/workflows/test.yml@refs/pull/67/merge
+GITHUB_WORKFLOW_SHA=400065561505a364ee842cba37a4a4ae92a6d6f8
+GITHUB_WORKSPACE=D:\a\Swifty-LLVM\Swifty-LLVM
 GOROOT_1_20_X64=C:\hostedtoolcache\windows\go\1.20.14\x64
 GOROOT_1_21_X64=C:\hostedtoolcache\windows\go\1.21.9\x64
 GOROOT_1_22_X64=C:\hostedtoolcache\windows\go\1.22.2\x64
@@ -423,9 +417,8 @@
 LIB=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\lib\x64;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\lib\um\x64;C:\Program Files (x86)\Windows Kits\10\lib\10.0.22621.0\ucrt\x64;C:\Program Files (x86)\Windows Kits\10\\lib\10.0.22621.0\\um\x64
 LIBPATH=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\ATLMFC\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\lib\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\lib\x86\store\references;C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.22621.0;C:\Program Files (x86)\Windows Kits\10\References\10.0.22621.0;C:\Windows\Microsoft.NET\Framework64\v4.0.30319
 llvm_package_basename=llvm-17.0.6-x86_64-unknown-windows-msvc17-MinSizeRel
-llvm_url_prefix=https://github.com/hylo-lang/llvm-build/releases/download/20240303-215025
 LOCALAPPDATA=C:\Users\runneradmin\AppData\Local
-LOGONSERVER=\\fv-az1111-280
+LOGONSERVER=\\fv-az972-783
 M2=C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin
 M2_REPO=C:\ProgramData\m2
 MAVEN_OPTS=-Xms256m
@@ -434,7 +427,7 @@
 npm_config_prefix=C:\npm\prefix
 NUMBER_OF_PROCESSORS=4
 OS=Windows_NT
-Path=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\VCPackages;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin\Roslyn;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\DiagnosticsHub\Collector;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Swift\runtime-development\usr\bin;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\Llvm\x64\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\Linux\bin\ConnectionManagerExe;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\vcpkg
+Path=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.38.33130\bin\HostX64\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\VCPackages;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin\Roslyn;C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\;C:\Program Files (x86)\HTML Help Workshop;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\FSharp\Tools;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Team Tools\DiagnosticsHub\Collector;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\CodeCoverage.Console;C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\\x64;C:\Program Files (x86)\Windows Kits\10\bin\\x64;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\\MSBuild\Current\Bin\amd64;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\Llvm\x64\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\Linux\bin\ConnectionManagerExe;C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\vcpkg
 PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
 PERFLOG_LOCATION_SETTING=RUNNER_PERFLOG
 PGBIN=C:\Program Files\PostgreSQL\14\bin
@@ -462,18 +455,16 @@
 RTOOLS44_HOME=C:\rtools44
 RUNNER_ARCH=X64
 RUNNER_ENVIRONMENT=github-hosted
-RUNNER_NAME=GitHub Actions 20
+RUNNER_NAME=GitHub Actions 17
 RUNNER_OS=Windows
 RUNNER_PERFLOG=C:\actions\perflog
 RUNNER_TEMP=D:\a\_temp
 RUNNER_TOOL_CACHE=C:\hostedtoolcache\windows
-RUNNER_TRACKING_ID=github_d03191ff-4661-46b9-aa98-c8242de0ec8c
-RUNNER_WORKSPACE=D:\a\hylo
+RUNNER_TRACKING_ID=github_f94d25e8-5833-4aca-b4b4-54d72cb05e59
+RUNNER_WORKSPACE=D:\a\Swifty-LLVM
 SBT_HOME=C:\Program Files (x86)\sbt\
 SDKROOT=C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Platforms\Windows.platform\Developer\SDKs\Windows.sdk
 SELENIUM_JAR_PATH=C:\selenium\selenium-server.jar
-spm-build-options=-Xswiftc -enable-testing --explicit-target-dependency-import-check error
-spm-test-options=--parallel
 STATS_BLT=true
 STATS_D=true
 STATS_D_D=true
@@ -492,8 +483,9 @@
 TMP=C:\Users\RUNNER~1\AppData\Local\Temp
 UCRTVersion=10.0.22621.0
 UniversalCRTSdkDir=C:\Program Files (x86)\Windows Kits\10\
-USERDOMAIN=fv-az1111-280
-USERDOMAIN_ROAMINGPROFILE=fv-az1111-280
+url_prefix=https://github.com/hylo-lang/llvm-build/releases/download/20240303-215025
+USERDOMAIN=fv-az972-783
+USERDOMAIN_ROAMINGPROFILE=fv-az972-783
 USERNAME=runneradmin
 USERPROFILE=C:\Users\runneradmin
 VCIDEInstallDir=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\VC\
@@ -524,10 +516,10 @@
 WIX=C:\Program Files (x86)\WiX Toolset v3.14\
 __DOTNET_ADD_64BIT=1
 __DOTNET_PREFERRED_BITNESS=64
-__VSCMD_PREINIT_PATH=C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Swift\runtime-development\usr\bin;C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps
+__VSCMD_PREINIT_PATH=C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.15.5\x64;C:\cabal\bin;C:\\ghcup\bin;C:\mingw64\bin;C:\Program Files\dotnet;C:\Program Files\MySQL\MySQL Server 8.0\bin;C:\Program Files\R\R-4.3.3\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\SeleniumWebDrivers\EdgeDriver\;C:\SeleniumWebDrivers\ChromeDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.21.9\x64\bin;C:\hostedtoolcache\windows\Python\3.9.13\x64\Scripts;C:\hostedtoolcache\windows\Python\3.9.13\x64;C:\hostedtoolcache\windows\Ruby\3.0.6\x64\bin;C:\Program Files\OpenSSL\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.412-8\x64\bin;C:\Program Files\ImageMagick-7.1.1-Q16-HDRI;C:\Program Files\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\ProgramData\docker-compose;C:\ProgramData\Chocolatey\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files\Microsoft SQL Server\160\DTS\Binn\;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.8.7\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps
 ##[endgroup]
 ##[group]Getting installed Swift version swift-5.9.2-RELEASE
 ##[endgroup]
 [command]C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swift.exe --version
-Swift version 5.9.2 (swift-5.9.2-RELEASE)
-Target: x86_64-unknown-windows-msvc
+##[error]The process 'C:\hostedtoolcache\windows\swift-5.9.2-RELEASE-windows10\5.9.2\x86_64\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swift.exe' failed with exit code 3221225781
+Cleaning up orphan processes
\ No newline at end of file
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -162,6 +162,9 @@ jobs:
         - os: macos-13
           swift: 'latest'
           development: false
+        - os: windows-latest
+          swift: '5.9'
+          development: false
         - os: ubuntu-latest
           swift: '5.3.0'
           development: false

--- a/src/installer/windows/installation/approach.ts
+++ b/src/installer/windows/installation/approach.ts
@@ -13,13 +13,21 @@ export async function firstDirectoryLayout(root?: string) {
   await fs.access(toolchain)
   const winsdk = path.join('Developer', 'SDKs', 'Windows.sdk')
   const sdkroot = path.join(devdir, 'Platforms', 'Windows.platform', winsdk)
-  const runtime = path.join(location, 'Swift', 'runtime-development')
+  const runtimeRoot = path.join(location, 'Swift')
+  const runtime = path.join(runtimeRoot, 'runtime-development')
+  const devPath = path.join(systemDrive, 'Program Files', 'Swift')
+  try {
+    await fs.access(devPath)
+    await fs.cp(devPath, runtimeRoot, {recursive: true})
+  } catch (error) {
+    core.debug(`Runtime check failed with "${error}"`)
+  }
   core.debug('First installation approach succeeded')
   return new Installation(location, toolchain, sdkroot, runtime, devdir)
 }
 
 export async function secondDirectoryLayout(root?: string) {
-  core.debug('Trying secong installation approach')
+  core.debug('Trying second installation approach')
   const systemDrive = root ?? process.env.SystemDrive ?? 'C:'
   const location = path.join(systemDrive, 'Program Files', 'Swift')
   const toolchainName = '0.0.0+Asserts'
@@ -28,9 +36,6 @@ export async function secondDirectoryLayout(root?: string) {
   const winsdk = path.join('Developer', 'SDKs', 'Windows.sdk')
   const sdkroot = path.join(location, 'Platforms', 'Windows.platform', winsdk)
   const runtime = path.join(location, 'Runtimes', '0.0.0')
-  const runtimeRoot = path.join(location, 'Swift')
-  const devPath = path.join(systemDrive, 'Program Files', 'Swift')
-  await fs.cp(devPath, runtimeRoot, {recursive: true})
   core.debug('Second installation approach succeeded')
   return new Installation(location, toolchain, sdkroot, runtime)
 }

--- a/src/installer/windows/installation/index.ts
+++ b/src/installer/windows/installation/index.ts
@@ -36,7 +36,7 @@ Installation.get = async (install?: string) => {
       core.debug(`Development directory at "${installation.devdir}"`)
       return installation
     } catch (error) {
-      core.debug(`Failed ${counter} time(s)`)
+      core.debug(`Failed ${counter} time(s), recent error "${error}"`)
     }
   }
   return undefined

